app/controllers/users_controller.rb

```diff
@@ -12,6 +11,11 @@ def index
   def show
   end
 
+  def me
+    @current_user
```

Comment:
One best practice in Rails is to limit to `an` instance variable per action. In this case I'm not sure of the reasons for the redundant call to `@current_user`
Also, you may not need to set the `@token` and `@current_user`!